### PR TITLE
Allow rendering new PDFs by changing `src` property after initial load

### PR DESCRIPTION
**Pull request by apalaniuk**

### CODE REVIEW

File: d2l-pdf-viewer.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
 				cursor: default;
 			}
 
-			d2l-page-load-progress[hidden] {
+			[hidden] {
```
Comment (ChrisKraljevicD2L) [inline d2l-pdf-viewer.html]: :+1:

File: d2l-pdf-viewer.html [hunk 2]
```diff
@@ -208,6 +228,16 @@
...
 				this._pageScale = this._pdfViewer.currentScale;
+			},
+			_onLoadStart: function() {
+				this._isLoading = true;
+				this.$.loadingBar.start();
+			},
+			_onLoadFinish: function() {
+				if (this._isLoading) {
+					this._isLoading = false;
+					this.$.loadingBar.finish();
```
Comment (ctwomey1) [inline d2l-pdf-viewer.html]: Is this command not idempotent or something?

Comment (apalaniuk) [reply d2l-pdf-viewer.html]: Functionally there should be no issue, but the component itself [doesn't keep track of loading status when applying styles](https://github.com/BrightspaceUI/page-load-progress/blob/master/d2l-page-load-progress.html#L126), so want to eg. make sure there are no unnecessary repaints in the case of multiple `pagerendered` events (granted, I did not investigate/measure). They would probably be scoped to this element anyway given how it's sized by the consumed (should be) so can remove if it really seems unnecessary.

File: d2l-pdf-viewer.html [hunk 3]
```diff
@@ -95,53 +94,75 @@
...
+			},
+			_onSrcChanged: function(src, isAttached) {
+				if (!src || !isAttached) {
+					return;
+				}
 
-				// Add event listeners before loading document
-				this._addEventListeners();
+				if (this._pdfDocument) {
+					this._pdfDocument.destroy();
```
Comment (apalaniuk) [inline d2l-pdf-viewer.html]: hm, I might have to `.destroy()` on the loading task as well.

File: d2l-pdf-viewer.html (hunk 2 above)
Comment (ctwomey1) [reply d2l-pdf-viewer.html]: it doesn't really matter, just seemed curious to me

**Comment by apalaniuk** [comment]
Will be implemented in a newer PR.

**Pull request closed without merging**